docs/self_supervised_models.md

```diff
@@ -0,0 +1,183 @@
+  data (30k scans, labels indicating presence of pneumonia),
+  [NIH Chest-Xray](https://www.kaggle.com/nih-chest-xrays/data) (112k Chest-Xray scans) or
+  [CheXpert](https://stanfordmlgroup.github.io/competitions/chexpert/) (228k scans).
+
+### Multi-dataset support
+
+During self-supervised training, a separate linear classifier is trained on top of learnt image embeddings. In this way,
+users can continuously monitor the representativeness of learnt image embeddings for a given downstream classification
+task. More importantly, the framework allows users to specify multiple datasets and data loaders for SimCLR/BYOL
+training and evaluation. For instance, a BYOL encoder can be learnt using a dataset that does not contain any target
+labels and embeddings can be evaluated throughout training on a separate dataset containing class labels. To enable this
+functionality, our SSLContainer takes two dataset names parameters: ``ssl_training_dataset_name`` to indicate which
+dataset to use for SSL training and ``classifier_dataset_name`` to indicate which dataset for the classification task
+used to monitor embeddings quality during training.
+
+## Quick start guide
+
+Here we described how to quickly start a training job with our ready made configs.
+
+### Example 1: training a SimCLR or BYOL model on CIFAR10
+
+To kick-off a training for a SimCLR and BYOL models on CIFAR10, simply run
+
+```
+python ML/runner.py --model=CIFAR10BYOL
+python ML/runner.py --model=CIFAR10SimCLR
+```
+
+For this dataset, it will automatically take care of downloading the dataset to your machine prior to starting training.
+
+### Example 2: training a BYOL model on Chest-Xray data
+
+#### Step 0: Get the data
+
+#### If you run on your local machine:
+
+Prior to starting training a model on this dataset, you will need to download it from Kaggle to your machine:
+
+* To use the RSNA Pneumonia Detection Challenge data: please download from
+  [here](https://www.kaggle.com/c/rsna-pneumonia-detection-challenge/data?select=stage_2_train_images). Make sure to
+  download all images and the `dataset.csv` file to your data folder. Please note that the labels are here merely used
+  for monitoring purposes.
+* To get the NIH dataset: please download from [here](https://www.kaggle.com/nih-chest-xrays/data). Make sure you
+  include also the csv files in your data folder, the code assumes the dataset files and all the images lie in your data
+  folder as downloaded from Kaggle. In particular, do not modify the original csv filenames (e.g. the code expects to
+  find the `Data_Entry_2017.csv` file within the data directory).
+* To get the CheXpert data: please download from [here](https://stanfordmlgroup.github.io/competitions/chexpert/), the
+  code assumes the dataset files and all the images lie in your data folder as downloaded.
+
+#### If you run on AML
+
+In order to train models on AML you will need to upload and register the datasets listed above to your storage account
```

Comment:
Uploading the dataset should be enough, right? It should be registered automatically.